tilings/tiling.py

```diff
@@ -56,6 +59,27 @@
 Cell = Tuple[int, int]
 ReqList = Tuple[GriddedPerm, ...]
 
+CellBasis = Dict[Cell, Tuple[List[Perm], List[Perm]]]
+Map = Dict[Cell, Cell]
```

Comment:
Should we call that `CellMap` to be more explicit?

Reply:
Agree.